platform-service-impl/src/main/java/esa/mo/platform/impl/provider/opssat/CameraOPSSATAdapter.java

```diff
@@ -129,7 +129,7 @@ private void initBSTCamera() throws IOException
 
   private synchronized void dumpHKTelemetry()
   {
-    bst_ims100_tele_std_t stdTM = new bst_ims100_tele_std_t();
+    final bst_ims100_tele_std_t stdTM = new bst_ims100_tele_std_t();
     ims100_api.bst_ims100_get_tele_std(stdTM);
     LOGGER.log(Level.INFO,
         String.format("Dumping HK Telemetry...\n"
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.

Reply:
Agree